org.contourterminal.Contour.yml

```diff
@@ -0,0 +1,106 @@
+app-id: org.contourterminal.Contour
+name: Contour Terminal Emulator
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: contour
+finish-args:
+  - --filesystem=host
+  - --device=all          # we need access to at least DRI (for OpenGL) & PTY
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --talk-name=org.freedesktop.Flatpak # Required for process `flatpak-spawn --host ...`.
+
+modules:
+  - name: contour
+    buildsystem: cmake-ninja
+    config-opts:
+      - "-DCONTOUR_INSTALL_TOOLS=ON"
+      - "-DCONTOUR_BLUR_PLATFORM_KWIN=ON"
+      - "-DCONTOUR_TESTING=OFF"
+      - "-DLIBTERMINAL_TESTING=OFF"
+    # NOTE: This post-install should be removed with the next release of Contour, when metainfo.xml is auto-installed.
+    post-install:
+      - install -p -m755 -d /app/share/metainfo
+      - install -p -m644 org.contourterminal.Contour.metainfo.xml /app/share/metainfo
+      - install -p -m644 org.contourterminal.Contour.desktop /app/share/applications
+      - rm /app/share/applications/contour.desktop
+      - mv /app/share/pixmaps/{contour,org.contourterminal.Contour}.png
```

Comment:
this icon location is invalid. And also we have `rename-icon` for that (but given it is in the wrong place, it's moot)